opentech/apply/users/templatetags/users_tags.py

```diff
@@ -0,0 +1,40 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.filter
+def backend_name(name):
```

Comment:
Is there much value in listing the backends we aren't supporting at this point in time?

Reply:
Probably not, but I like to future-proof things.
Will remove all but Google/Facebook/Twitter

Reply:
ok, removed all